packages/core/src/StellarRequest.js

```diff
@@ -52,22 +58,30 @@ export default class StellarRequest extends StellarCore {
 
   setMiddlewares() {
     const me = this;
-    this.requestMiddlewares = this.handlerChain.concat(
-      {
-        fn(req) {
-          return me.transport
-            .request(req)
-            .catch(error => (error.__stellarResponse ? error.__stellarResponse : me._prepareResponse(req, error)))
-            .then(res => responseHandler(res));
-        },
-      });
 
-    this.fireAndForgetMiddlewares = this.handlerChain.concat(
-      {
-        fn(request) {
-          return me.transport.fireAndForget(request);
-        },
-      });
+    function buildFinalMws(dispatchFn) {
+      if (me.optimizeLocalHandlers) {
+        return [
+          { fn: mwLocalDispatchFactory({ stringifyDates: me.stringifyDates }) },
```

Comment:
i guess it's fine for now, but it's too much trouble.. we can put it in `config-stellar.js` in the project

Reply:
has to be the second last one. only way i can guarantee that is by putting it here